crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -2066,6 +2080,15 @@ impl<'db> TypeInferenceBuilder<'db> {
                 }
             }
 
+            (Type::LiteralString, Type::IntLiteral(n), ast::Operator::Mult)
+            | (Type::IntLiteral(n), Type::LiteralString, ast::Operator::Mult) => {
+                if n < 1 {
+                    Type::StringLiteral(StringLiteralType::new(self.db, Box::default()))
+                } else {
+                    Type::LiteralString
+                }
```

Comment:
Can we add tests for these two new branches? (I realize it's a little irritating to create a `LiteralString` in a test, since at the moment it requires creating a very long string literal -- but it's not _too_ bad.)

Reply:
Done!

Reply:
Thank you! I'm seeing a new test for the new match arm around adding Literal strings, but not for these two cases of multiplying by a literal int?

Reply:
Whoops, good catch! Added now